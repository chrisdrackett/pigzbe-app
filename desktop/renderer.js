import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
