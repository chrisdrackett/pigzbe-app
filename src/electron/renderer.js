import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
