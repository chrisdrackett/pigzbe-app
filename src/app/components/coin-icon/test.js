import React from 'react';
import renderer from 'react-test-renderer';
import CoinIcon from './';

const props = {
    coin: 'btc',
    style: null
};

describe('CoinIcon', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<CoinIcon {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
