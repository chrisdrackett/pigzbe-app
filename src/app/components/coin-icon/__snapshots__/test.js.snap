// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CoinIcon renders correctly 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../src/app/components/coin-icon/images/coin-btc.png",
    }
  }
  style={
    Object {
      "height": 35,
      "width": 35,
    }
  }
/>
`;
